packages/react-components/src/components/gif.tsx

```diff
@@ -1,11 +1,15 @@
 import { giphyBlue, giphyGreen, giphyPurple, giphyRed, giphyYellow } from '@giphy/js-brand'
 import { IGif, IUser } from '@giphy/js-types'
 import { checkIfWebP, getAltText, getBestRenditionUrl, getGifHeight, Logger } from '@giphy/js-util'
+import moat from '@giphy/moat-loader'
 import { css, cx } from 'emotion'
 import React, { ReactType, SyntheticEvent, useEffect, useRef, useState } from 'react'
+import constructMoatData from '../util/construct-moat-data'
 import * as pingback from '../util/pingback'
 import AdPill from './ad-pill'
 
+const moatLoader = moat.loadMoatTag('giphydisplay879451385633')
```

Comment:
@cosmocochrane1 is this right?

Reply:
@cosmocochrane1 it's not. did we update moat loader to use our bucket script?

Reply:
we're waiting on moat to send us the new dynamic loader that pulls from our moattag

Reply:
for now this is what they said to use